recipes/ffmpeg/all/conanfile.py

```diff
@@ -1,18 +1,21 @@
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
-from conan.tools.apple import is_apple_os
+from conan.tools.apple import fix_apple_shared_install_name, is_apple_os, to_apple_arch, XCRun
 from conan.tools.build import cross_building
-from conan.tools.files import get, rename, rmdir
+from conan.tools.files import chdir, copy, get, rename, replace_in_file, rm, rmdir
+from conan.tools.gnu.get_gnu_triplet import _get_gnu_triplet
```

Comment:
@MartinDelille no need to trigger c3i again if we know linter will always prevent merge due to this import of a private function.

Reply:
Done here: https://github.com/conan-io/conan-center-index/pull/15830